zeroae/rocksdb/c/c.i

```diff
@@ -1,29 +1,38 @@
 %{
 #define SWIG_FILE_WITH_INIT
 #include <rocksdb/c.h>
+
+// TODO: This makes the interface non-reentrant
+char* ERRMSG = NULL;
```

Comment:
As the node implies, the code is not thread safe because of this `ERRMSG` local variable.